src/transformers/modeling_utils.py

```diff
@@ -1845,10 +1845,18 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                     " bitsandbytes-cuda113` for CUDA 11.3)"
                 )
             if torch_dtype == "auto" or torch_dtype != torch.float16:
-                torch_dtype = (
-                    torch.float16
-                )  # We force the `dtype` to be float16, this is a requirement from `bitsandbytes`
+                torch_dtype = torch.float16  # We force the `dtype` to be float16, this is a requirement from `bitsandbytes`
                 logger.info("Loading the model in mixed int8 - forcing the weights to be casted in float16")
+            if device_map is None:
+                raise ValueError(
+                    "A device map needs to be passed to run convert models into mixed-int8 format. Please run"
+                    "`.from_pretrained` with `device_map='auto'`"
+                )
+            if (from_tf | from_flax):
```

Comment:
```suggestion
            if (from_tf or from_flax):
```
Let's use the English version which is more readable ;-)

Reply:
Yeah ahah thanks for the suggestion!

Reply:
I could not commit the suggestion from the GUI, therefore pushed a commit here: f6eb9456a8459cec5774bfa22b377c7e0ffc859f